reconcile/utils/terrascript_client.py

```diff
@@ -1781,16 +1783,24 @@ def populate_tf_resource_role(self, resource, namespace_info):
         self.init_common_outputs(tf_resources, namespace_info, output_prefix,
                                  output_resource_name, annotations)
 
+        assume_users = resource['assume_users']
+        if assume_users:
+            assume_principal = {
+                "AWS": assume_users
+            }
+        else:
```

Comment:
Why one or the other? Can the principal be a list and we can allow both aws and service?